test/e2e/egress_services.go

```diff
@@ -1160,7 +1160,9 @@ spec:
 				gomega.Consistently(func() error {
 					for _, pod := range append(net1.createdPods, net2.createdPods...) {
 						err := curlAgnHostClientIPFromPod(f.Namespace.Name, pod, "", dst, podHTTPPort)
-						if err != nil && strings.Contains(err.Error(), "exit code 28") {
+						if err != nil && (strings.Contains(err.Error(), fmt.Sprintf("exit code 28")) ||
+							// github runners don't have any routes for IPv6, so we get CURLE_COULDNT_CONNECT
```

Comment:
hehe the level of detail one has to go to! Can you provide source of how you found this? (both the github runner limitation and the source code 7 versus 28)